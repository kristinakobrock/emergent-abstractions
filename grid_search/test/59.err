  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  8%|#######4                                                                                     | 4/50 [00:00<00:01, 32.25it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 28.56it/s] 22%|####################2                                                                       | 11/50 [00:00<00:01, 27.11it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 23.93it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 25.64it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 23.85it/s] 48%|############################################1                                               | 24/50 [00:00<00:01, 25.42it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 23.45it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 24.82it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 24.99it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 26.24it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 27.10it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 25.16it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 26.69it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 27.11it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.79it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 28.46it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 24.78it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 18.50it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 20.69it/s]100%|############################################################################################| 13/13 [00:00<00:00, 21.66it/s]
