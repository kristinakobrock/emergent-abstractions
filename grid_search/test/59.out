# ["--attributes=5", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.683777928352356, "acc": 0.5998473763465881, "length": 3.455045223236084, "mode": "train", "epoch": 1}
{"loss": 0.686983585357666, "acc": 0.556445300579071, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6630279421806335, "acc": 0.6219017505645752, "length": 3.035677194595337, "mode": "train", "epoch": 2}
{"loss": 0.6691774129867554, "acc": 0.6085938811302185, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6586254239082336, "acc": 0.6270542144775391, "length": 2.415440082550049, "mode": "train", "epoch": 3}
{"loss": 0.6650481820106506, "acc": 0.6179686188697815, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6535784602165222, "acc": 0.6277907490730286, "length": 2.460314989089966, "mode": "train", "epoch": 4}
{"loss": 0.6474457383155823, "acc": 0.6445373296737671, "length": 1.52734375, "mode": "test", "epoch": 4}
{"loss": 0.6313632130622864, "acc": 0.6452637910842896, "length": 2.612910509109497, "mode": "train", "epoch": 5}
{"loss": 0.633472204208374, "acc": 0.6559142470359802, "length": 2.3828125, "mode": "test", "epoch": 5}
{"loss": 0.6098517179489136, "acc": 0.6687020659446716, "length": 2.6615993976593018, "mode": "train", "epoch": 6}
{"loss": 0.6434086561203003, "acc": 0.6536590456962585, "length": 2.484375, "mode": "test", "epoch": 6}
{"loss": 0.5956577062606812, "acc": 0.6790635585784912, "length": 3.137282609939575, "mode": "train", "epoch": 7}
{"loss": 0.6237772703170776, "acc": 0.6727294921875, "length": 3.046875, "mode": "test", "epoch": 7}
{"loss": 0.5843283534049988, "acc": 0.6925618648529053, "length": 3.2614455223083496, "mode": "train", "epoch": 8}
{"loss": 0.6033095121383667, "acc": 0.6862426400184631, "length": 3.13671875, "mode": "test", "epoch": 8}
{"loss": 0.5648220777511597, "acc": 0.7076717019081116, "length": 3.1857643127441406, "mode": "train", "epoch": 9}
{"loss": 0.6075820326805115, "acc": 0.6955780982971191, "length": 3.40234375, "mode": "test", "epoch": 9}
{"loss": 0.5513512492179871, "acc": 0.7185133695602417, "length": 3.4981188774108887, "mode": "train", "epoch": 10}
{"loss": 0.6655751466751099, "acc": 0.677594006061554, "length": 3.6328125, "mode": "test", "epoch": 10}
{"loss": 0.5410683155059814, "acc": 0.723356306552887, "length": 3.55538272857666, "mode": "train", "epoch": 11}
{"loss": 0.6227723360061646, "acc": 0.7121337056159973, "length": 3.7265625, "mode": "test", "epoch": 11}
{"loss": 0.5271623134613037, "acc": 0.7386898398399353, "length": 3.5544214248657227, "mode": "train", "epoch": 12}
{"loss": 0.5929601192474365, "acc": 0.723919689655304, "length": 3.6953125, "mode": "test", "epoch": 12}
{"loss": 0.505792498588562, "acc": 0.7475619316101074, "length": 3.6019601821899414, "mode": "train", "epoch": 13}
{"loss": 0.5685909986495972, "acc": 0.7358947396278381, "length": 3.76953125, "mode": "test", "epoch": 13}
{"loss": 0.4877778887748718, "acc": 0.7618590593338013, "length": 3.661917209625244, "mode": "train", "epoch": 14}
{"loss": 0.5489356517791748, "acc": 0.7612395882606506, "length": 3.796875, "mode": "test", "epoch": 14}
{"loss": 0.47278115153312683, "acc": 0.7710021734237671, "length": 3.622765302658081, "mode": "train", "epoch": 15}
{"loss": 0.543962836265564, "acc": 0.7470886707305908, "length": 3.671875, "mode": "test", "epoch": 15}
{"loss": 0.45734816789627075, "acc": 0.784967303276062, "length": 3.6933443546295166, "mode": "train", "epoch": 16}
{"loss": 0.5298025608062744, "acc": 0.7531830072402954, "length": 3.8125, "mode": "test", "epoch": 16}
{"loss": 0.4406810402870178, "acc": 0.7971557974815369, "length": 3.799013376235962, "mode": "train", "epoch": 17}
{"loss": 0.5310784578323364, "acc": 0.7709747552871704, "length": 3.890625, "mode": "test", "epoch": 17}
{"loss": 0.4305378794670105, "acc": 0.8030494451522827, "length": 3.7729923725128174, "mode": "train", "epoch": 18}
{"loss": 0.5028492212295532, "acc": 0.777587890625, "length": 3.92578125, "mode": "test", "epoch": 18}
{"loss": 0.40958115458488464, "acc": 0.8112033605575562, "length": 3.7947027683258057, "mode": "train", "epoch": 19}
{"loss": 0.49956467747688293, "acc": 0.8026031255722046, "length": 3.875, "mode": "test", "epoch": 19}
{"loss": 0.3902314007282257, "acc": 0.8250629305839539, "length": 3.776796340942383, "mode": "train", "epoch": 20}
{"loss": 0.45365944504737854, "acc": 0.8109680414199829, "length": 3.86328125, "mode": "test", "epoch": 20}
{"loss": 0.37965941429138184, "acc": 0.8303632736206055, "length": 3.8151166439056396, "mode": "train", "epoch": 21}
{"loss": 0.4680976867675781, "acc": 0.8128600716590881, "length": 3.88671875, "mode": "test", "epoch": 21}
{"loss": 0.37320783734321594, "acc": 0.8315047025680542, "length": 3.7987048625946045, "mode": "train", "epoch": 22}
{"loss": 0.4666912853717804, "acc": 0.8052337765693665, "length": 3.8046875, "mode": "test", "epoch": 22}
{"loss": 0.3715841770172119, "acc": 0.8339446187019348, "length": 3.8172388076782227, "mode": "train", "epoch": 23}
{"loss": 0.4635261595249176, "acc": 0.803149402141571, "length": 3.82421875, "mode": "test", "epoch": 23}
{"loss": 0.36550092697143555, "acc": 0.8383082747459412, "length": 3.782545566558838, "mode": "train", "epoch": 24}
{"loss": 0.45208877325057983, "acc": 0.8127594590187073, "length": 3.75390625, "mode": "test", "epoch": 24}
{"loss": 0.3574153780937195, "acc": 0.8397464156150818, "length": 3.814889430999756, "mode": "train", "epoch": 25}
{"loss": 0.46353578567504883, "acc": 0.8065032362937927, "length": 3.796875, "mode": "test", "epoch": 25}
{"loss": 0.3455037772655487, "acc": 0.8435128331184387, "length": 3.825528144836426, "mode": "train", "epoch": 26}
{"loss": 0.4531007409095764, "acc": 0.8119568824768066, "length": 3.828125, "mode": "test", "epoch": 26}
{"loss": 0.3455907702445984, "acc": 0.8466572165489197, "length": 3.8221566677093506, "mode": "train", "epoch": 27}
{"loss": 0.45433947443962097, "acc": 0.8108673095703125, "length": 3.80078125, "mode": "test", "epoch": 27}
{"loss": 0.33594971895217896, "acc": 0.8489128947257996, "length": 3.8441944122314453, "mode": "train", "epoch": 28}
{"loss": 0.4664209485054016, "acc": 0.8160156011581421, "length": 3.875, "mode": "test", "epoch": 28}
{"loss": 0.3425663411617279, "acc": 0.8505346179008484, "length": 3.8394875526428223, "mode": "train", "epoch": 29}
{"loss": 0.4451000392436981, "acc": 0.8160064220428467, "length": 3.86328125, "mode": "test", "epoch": 29}
{"loss": 0.33946123719215393, "acc": 0.8499742746353149, "length": 3.8562216758728027, "mode": "train", "epoch": 30}
{"loss": 0.44496822357177734, "acc": 0.822338879108429, "length": 3.8828125, "mode": "test", "epoch": 30}
{"loss": 0.33124303817749023, "acc": 0.852903425693512, "length": 3.8476712703704834, "mode": "train", "epoch": 31}
{"loss": 0.444461852312088, "acc": 0.8235107660293579, "length": 3.84375, "mode": "test", "epoch": 31}
{"loss": 0.3291655480861664, "acc": 0.8557260036468506, "length": 3.8373019695281982, "mode": "train", "epoch": 32}
{"loss": 0.4594799876213074, "acc": 0.8171660900115967, "length": 3.90625, "mode": "test", "epoch": 32}
{"loss": 0.32298797369003296, "acc": 0.8541438579559326, "length": 3.8851730823516846, "mode": "train", "epoch": 33}
{"loss": 0.4328176975250244, "acc": 0.8214904069900513, "length": 3.84375, "mode": "test", "epoch": 33}
{"loss": 0.3199402391910553, "acc": 0.8564807772636414, "length": 3.8431708812713623, "mode": "train", "epoch": 34}
{"loss": 0.42783665657043457, "acc": 0.8196990489959717, "length": 3.89453125, "mode": "test", "epoch": 34}
{"loss": 0.31818559765815735, "acc": 0.8564264178276062, "length": 3.8904788494110107, "mode": "train", "epoch": 35}
{"loss": 0.43388187885284424, "acc": 0.8231995105743408, "length": 3.859375, "mode": "test", "epoch": 35}
{"loss": 0.31497833132743835, "acc": 0.8601478934288025, "length": 3.8311800956726074, "mode": "train", "epoch": 36}
{"loss": 0.4259326457977295, "acc": 0.8263427019119263, "length": 3.87109375, "mode": "test", "epoch": 36}
{"loss": 0.3077419102191925, "acc": 0.8654589056968689, "length": 3.875013589859009, "mode": "train", "epoch": 37}
{"loss": 0.4263935685157776, "acc": 0.8227050304412842, "length": 3.88671875, "mode": "test", "epoch": 37}
{"loss": 0.3004760444164276, "acc": 0.8662412762641907, "length": 3.8637869358062744, "mode": "train", "epoch": 38}
{"loss": 0.4041297137737274, "acc": 0.8352416753768921, "length": 3.8828125, "mode": "test", "epoch": 38}
{"loss": 0.295419842004776, "acc": 0.8706014156341553, "length": 3.8951151371002197, "mode": "train", "epoch": 39}
{"loss": 0.4261050224304199, "acc": 0.8348662853240967, "length": 3.91015625, "mode": "test", "epoch": 39}
{"loss": 0.29293638467788696, "acc": 0.8753894567489624, "length": 3.8846964836120605, "mode": "train", "epoch": 40}
{"loss": 0.41079181432724, "acc": 0.8313843011856079, "length": 3.90625, "mode": "test", "epoch": 40}
{"loss": 0.29041340947151184, "acc": 0.8764500617980957, "length": 3.8813252449035645, "mode": "train", "epoch": 41}
{"loss": 0.41505008935928345, "acc": 0.8366822004318237, "length": 3.921875, "mode": "test", "epoch": 41}
{"loss": 0.2809678018093109, "acc": 0.8763545751571655, "length": 3.889427661895752, "mode": "train", "epoch": 42}
{"loss": 0.43322083353996277, "acc": 0.825213611125946, "length": 3.921875, "mode": "test", "epoch": 42}
{"loss": 0.2803844213485718, "acc": 0.8829898834228516, "length": 3.9075891971588135, "mode": "train", "epoch": 43}
{"loss": 0.41634097695350647, "acc": 0.8313354253768921, "length": 3.91796875, "mode": "test", "epoch": 43}
{"loss": 0.27300262451171875, "acc": 0.8858317732810974, "length": 3.894906759262085, "mode": "train", "epoch": 44}
{"loss": 0.41090357303619385, "acc": 0.8300689458847046, "length": 3.91796875, "mode": "test", "epoch": 44}
{"loss": 0.26045697927474976, "acc": 0.8920853137969971, "length": 3.8899176120758057, "mode": "train", "epoch": 45}
{"loss": 0.40676742792129517, "acc": 0.841171383857727, "length": 3.91015625, "mode": "test", "epoch": 45}
{"loss": 0.25616738200187683, "acc": 0.8947461843490601, "length": 3.907288074493408, "mode": "train", "epoch": 46}
{"loss": 0.40087398886680603, "acc": 0.844067394733429, "length": 3.90234375, "mode": "test", "epoch": 46}
{"loss": 0.24845214188098907, "acc": 0.8985869288444519, "length": 3.9001824855804443, "mode": "train", "epoch": 47}
{"loss": 0.3996439576148987, "acc": 0.8448913097381592, "length": 3.90234375, "mode": "test", "epoch": 47}
{"loss": 0.24534232914447784, "acc": 0.9022799134254456, "length": 3.8750689029693604, "mode": "train", "epoch": 48}
{"loss": 0.4079725444316864, "acc": 0.8472596406936646, "length": 3.91796875, "mode": "test", "epoch": 48}
{"loss": 0.2438059002161026, "acc": 0.9025043249130249, "length": 3.9027040004730225, "mode": "train", "epoch": 49}
{"loss": 0.4155091643333435, "acc": 0.8448333740234375, "length": 3.90234375, "mode": "test", "epoch": 49}
{"loss": 0.24091997742652893, "acc": 0.9047442078590393, "length": 3.9100515842437744, "mode": "train", "epoch": 50}
{"loss": 0.39213433861732483, "acc": 0.8452422618865967, "length": 3.8984375, "mode": "test", "epoch": 50}
{"loss": 0.23424141108989716, "acc": 0.9073421955108643, "length": 3.898791790008545, "mode": "train", "epoch": 51}
{"loss": 0.3903065025806427, "acc": 0.8499603271484375, "length": 3.90234375, "mode": "test", "epoch": 51}
{"loss": 0.22652246057987213, "acc": 0.9123527407646179, "length": 3.893190860748291, "mode": "train", "epoch": 52}
{"loss": 0.37909191846847534, "acc": 0.8547363877296448, "length": 3.9375, "mode": "test", "epoch": 52}
{"loss": 0.22362510859966278, "acc": 0.9139105081558228, "length": 3.907505512237549, "mode": "train", "epoch": 53}
{"loss": 0.3848612606525421, "acc": 0.8572357892990112, "length": 3.9140625, "mode": "test", "epoch": 53}
{"loss": 0.2206399291753769, "acc": 0.9164249300956726, "length": 3.903059244155884, "mode": "train", "epoch": 54}
{"loss": 0.3894493877887726, "acc": 0.8546813726425171, "length": 3.95703125, "mode": "test", "epoch": 54}
{"loss": 0.2200353443622589, "acc": 0.9150474667549133, "length": 3.921441078186035, "mode": "train", "epoch": 55}
{"loss": 0.38385671377182007, "acc": 0.8581116199493408, "length": 3.9375, "mode": "test", "epoch": 55}
{"loss": 0.20923109352588654, "acc": 0.920414924621582, "length": 3.906973123550415, "mode": "train", "epoch": 56}
{"loss": 0.3894002437591553, "acc": 0.8616394400596619, "length": 3.9453125, "mode": "test", "epoch": 56}
{"loss": 0.2065660059452057, "acc": 0.9223194718360901, "length": 3.935553789138794, "mode": "train", "epoch": 57}
{"loss": 0.38287821412086487, "acc": 0.8659026622772217, "length": 3.93359375, "mode": "test", "epoch": 57}
{"loss": 0.20794448256492615, "acc": 0.92458575963974, "length": 3.915919542312622, "mode": "train", "epoch": 58}
{"loss": 0.38163912296295166, "acc": 0.8692932724952698, "length": 3.91015625, "mode": "test", "epoch": 58}
{"loss": 0.20127816498279572, "acc": 0.9228580594062805, "length": 3.9324586391448975, "mode": "train", "epoch": 59}
{"loss": 0.38832029700279236, "acc": 0.8621764779090881, "length": 3.8984375, "mode": "test", "epoch": 59}
{"loss": 0.19646285474300385, "acc": 0.9278470277786255, "length": 3.917625665664673, "mode": "train", "epoch": 60}
{"loss": 0.3700867295265198, "acc": 0.8730682730674744, "length": 3.96875, "mode": "test", "epoch": 60}
test accuracy: 0.7852821946144104
